src/jukebox/components/rfid/reader/__init__.py

```diff
@@ -38,7 +39,10 @@ def register(self, func: Callable[[str, int], None]):
             :noindex:
 
             :param card_id: Card ID
-            :param state: 0 if card id is registered, 1 if card id is unknown
+            :param state:
+                -1  on receiving card id (before processing card entries),
+                0   if card id is registered,
+                1   if card id is unknown
```

Comment:
Refactored this to have the state as enum to have defined entries (instead of magic string, int, ...).

Used this also in playermpd with the addition that the Callback class has now a generic for state which is given on construction. This way only a new state enum must be added to add a new callback for different play-functions (but maybe just to much foreseeing?)

What do you think about this solution?